src/main/java/zpi/attraction/entity/Attraction.java

```diff
@@ -0,0 +1,61 @@
+package zpi.attraction.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+@Entity
+@Table(name = "attractions")
+public class Attraction {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "district_id")
+    private Integer districtId;
+
+    @Column(name = "district_name")
+    private String districtName;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "attraction_type")
+    private String attractionType;
```

Comment:
To przydało by się też jakoś wydzielić. Myślę, że nawet całkowite wydzielenie tego do innej tabeli zamiast stworzenia enuma będzie spoko